tests/helpers.py

```diff
@@ -71,6 +72,16 @@ class Meta:
     user = factory.SubFactory(UserFactory)
 
 
+class SearchModelFactory(factory.django.DjangoModelFactory):
```

Comment:
Might not be related to this PR, but the `Generate API Key` button in the config page also has some issue that causes it to not return the newly generated key.

Reply:
This was resolved by Saba